docs/Homebrew-Governance.md

```diff
@@ -24,7 +26,7 @@
 
 ## 3. General Meetings of Members
 
-1. A general meeting of the members may be called by the PLC with at least three weeks notice.
+1. A general meeting of the members may be called by either an ordinary resolution of the PLC or a majority of the entire membership. The membership must be given at least three weeks notice of a general meeting.
```

Comment:
Is "a majority of the entire membership" the same as "an ordinary resolution of the entire membership"?

We use both "majority vote" and "ordinary resolution" in this document. If there's a difference, it's not very clear to me what it is.

Even if there's a difference, I'm not sure how consistent we're being in this document. For example, why is a resolution needed to remove a PLC member a "majority vote" is needed for the PLC to add a new member?

For example, the PLC can remove a TSC member with an "ordinary resolution" (7.5) but can add a new member with a "majority vote" (2.1). To me, those two situations seem similar enough to warrant the same procedure. It feels to me like we're just being inconsistent, so I recommend either clarifying the difference or being more consistent throughout the document.

Reply:
👍 for making the terms consistent throughout the document.

Reply:
IANAL, but my understanding is that "ordinary resolution" typically refers to a formal document (the "resolution") laying out the matter to be voted on, and is passed by a majority vote. "Survivor" tribal councils go by majority vote, not ordinary resolutions.

Therefore:
> A general meeting of the members may be called by either an ordinary resolution of the PLC or a majority of the entire membership.

is perfectly correct _if_ the PLC actually creates an official document (that can be archived for posterity) calling for the meeting, then votes on it. I imagine the alternative would be executed along the lines of "Hi everyone, we're proposing a GM at <timestamp>, you OK with that?"

Reply:
> For example, why is a resolution needed to remove a PLC member a "majority vote" is needed for the PLC to add a new member?

Perhaps because forcibly terminating membership is a serious matter that requires extra care and formality.

Reply:
That distinction makes sense. I would think, though, that most things that are voted on are based on a formal document. Maybe that's a PR (like this) or a message in Slack, or something else. I guess I also don't know what counts as "formula" also. This coming meeting will be my first GM so I haven't actually experienced this.

As to your example, I would say that either that should count as "formal enough" if it's a message in slack that can be referenced in the future or we don't do those kinds of informal votes. I'm not PLC or a lawyer, or course.

I guess my point is that it doesn't feel like the actions that use "majority vote" are any less official or significantly different from those that use "ordinary resolution." Maybe someone who's been to one before can clarify that for me (@sjackman maybe you can weigh in?)

Reply:
>> A majority of the entire membership of the PLC is required to pass an ordinary resolution.

>  Is it necessary to include this given the new definition for an "ordinary resolution"?

Yes, because it requires a majority of the _entire_ membership of the PLC, rather than a majority of the votes cast, which is the usual case for an ordinary resolution.

Reply:
> We use both "majority vote" and "ordinary resolution" in this document. If there's a difference, it's not very clear to me what it is.

There is no difference. I've changed "majority vote" to "ordinary resolution", except for elections, which are not themselves a resolution.

Reply:
> Two comments about this. First, doesn't this belong in section 4 ("Project Leadership Committee") rather than section 5 ("Meetings of the Project Leadership Committee")? I don't think it's not really about meetings. This isn't directly related to the other changes in this PR so it may not be appropriate here.

🤷‍♂️ The dispute would be settled by a resolution of the PLC, so my guess is that's why it's here in meetings of the PLC.

> Second, should the "with compromise" phrase that was added to item 7.3 (which is essentially the TSC equivalent of 5.6) be added here as well?

👍 I've added "with compromised".

Reply:
Thanks, @sjackman!

In my mind, moving item 5.6 to section 4 makes more sense, but I don't feel strongly about it. If desired, I can make a PR in the governance repo explaining my reasoning. Otherwise, I'm happy to leave it as is.